ui/src/app/pages/profile/profile-page.tsx

```diff
@@ -8,10 +8,12 @@ import {Button, Clickable} from 'app/components/buttons';
 import { FlexColumn, FlexRow } from 'app/components/flex';
```

Comment:
For any PR, I usually flag any files with notable changes but no additional unit tests.

Are there any new tests that might be worth adding to profile-page.spec? Given that most of the new functionality is well-encapsulated by demographics-survey, it might be a good use case for Enzyme shallow-rendering, just to test / verify that we've correctly hooked up the component callbacks and are handling API responses, etc.

Reply:
I have now added tests